mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -78,6 +85,7 @@ pub enum UtxoCoinBuildError {
         mode: String,
     },
     InvalidPathToAddress(String),
+    WalletConnectError(String),
```

Comment:
How about keeping it strongly typed?

```suggestion
    WalletConnectError(WalletConnectError),
```